lib/ansible/module_utils/basic.py

```diff
@@ -707,7 +707,7 @@ def __init__(self, argument_spec, bypass_checks=False, no_log=False,
         except Exception:
             e = get_exception()
             # Use exceptions here because it isn't safe to call fail_json until no_log is processed
-            print('\n{"failed": true, "msg": "Module alias error: %s"}' % str(e))
+            print('\n{"failed": true, "msg": "Module alias error: %s"}' % e)
```

Comment:
Well, I wasn't really fixing anything in this specific case. Just that `%s` implicitly converts to string.
But if `to_native()` is preferred for exceptions, I don't mind fixing it everywhere too.